IBAnimatable/SliderImagesDesignable.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  Created by phimage on 25/03/2017.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import Foundation
+
+import UIKit
+
+/// Protocol for designing slider image
+public protocol SliderImagesDesignable {
+
+  /**
+   * The thumb image when slider state is `normal`
+   */
+  var thumbImage: UIImage? { get set }
+
+  /**
+   * The thumb image when slider state is `highlighted`
+   */
+  var thumbHighlightedImage: UIImage? { get set }
+
+  /**
+   * The minimum track image when slider state is `normal`
+   */
+  var minimumTrackImage: UIImage? { get set }
+
+  /**
+   * The minimum track image when slider state is `highlighted`
+   */
+  var minimumTrackHighlightedImage: UIImage? { get set }
+
+  /**
+   * The maximum track image when slider state is `normal`
+   */
+  var maximumTrackImage: UIImage? { get set }
+
+  /**
+   * The maximum track image when slider state is `highlighted`
+   */
+  var maximumTrackHighlightedImage: UIImage? { get set }
+
+}
+
+extension SliderImagesDesignable where Self: UISlider {
```

Comment:
I think we need to make it `public` for the others can use those extension methods outside of `IBAnimatable` framework.